requirements_roles.yml

```diff
@@ -25,7 +25,3 @@
 - src: https://github.com/ARTbio/ansible-galaxy-tools.git
   name: galaxyprojectdotorg.galaxy-tools
   version: galaxykickstart
-
```

Comment:
Is this not necessary anymore because you're using CVMFS for tool dependencies or because `conda_auto_init` is set to True ? In general this should be more robust than letting Galaxy install Conda when running multiple Galaxy processes. Especially if the first startup had been killed for some reason and a lock file had been left behind.

Reply:
This is marking the first changed line that removes the miniconda role.

Reply:
Ah OK. back to your question, it is true that `conda_auto_init` is set to True. This is the default Galaxy config:

```
  # Set to True to instruct Galaxy to install Conda from the web
  # automatically if it cannot find a local copy and conda_exec is not
  # configured.
  #conda_auto_init: true
```